ports/geckolib/gecko_bindings/structs_debug.rs

```diff
@@ -3918,7 +3880,43 @@ fn bindgen_test_layout_nsFont() {
 }
 #[repr(i8)]
 #[derive(Debug, Copy, Clone, Eq, PartialEq, Hash)]
+pub enum StyleBasicShape { Polygon = 0, Circle = 1, Ellipse = 2, Inset = 3, }
```

Comment:
This file is autogenerated 😄 

(and it's common style. Makes it easy to add more without perturbing diffs too much)